README.md

```diff
@@ -14,6 +14,21 @@ To use this bot, you need to do the following:
 5. Install the required Python packages by running `pip install -r requirements.txt.`
 6. Run the `main.py` file to start the bot.
 
+## Setup for Development
+1. Clone this repository to your local machine.
+2. Modify `config.py` with your settings.
+3. Add the required secrets to `.env.example`
+4. Rename `.env.example` to `.env`
+5. Install `poetry` using `pip install poetry`
```

Comment:
don't recommend pip installation. The official [docs](https://python-poetry.org/docs/#installation) state to use the official installer unless you know what you're doing.

Reply:
Ok will update